extensions/cornerstone-dicom-sr/src/commandsModule.js

```diff
@@ -37,6 +37,12 @@ const _generateReport = (
   // Add in top level series options
   Object.assign(dataset, options);
 
+  // Set the default character set as UTF-8
+  // https://dicom.innolitics.com/ciods/nm-image/sop-common/00080005
+  if (typeof dataset.SpecificCharacterSet === 'undefined') {
+    dataset.SpecificCharacterSet = 'ISO_IR 192';
```

Comment:
any chance for this dataset.SpecificCharacterSet to be null?

Reply:
It will always be null because it isn't one of the fields copied from anywhere else.  Not saying it can't be provided in the future, but right now it will get into the if.